src/components/LoginPage/LoginPage.js

```diff
@@ -1,57 +1,78 @@
 import { Col, Button, Row, Container, Card, Form } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import React, { useState } from 'react';
+import UserManager from "../admin_panel/utils/UserManager";
 
 export default function LoginPage() {
+  const [login, setLogin] = useState('');
+  const [password, setPassword] = useState('');
+  const [message, setMessage] = useState('');
+  const userManager = new UserManager();
+  
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const users = userManager.getUsers();
+    const user = users.find((u) => u.login === login && u.password === password);
```

Comment:
1. zmiana nazwy by się tu przydała jako że to jest boolean np. `isLoginSuccess`
2. 13 i 14 linia może być zamieniona w jedną:
`const isLoginSuccess = userManager.getUsers().find((u) => u.login === login && u.password === password);`